lib/scanner.js

```diff
@@ -1536,7 +1536,8 @@
       'N': '\x85',
       '_': '\xA0',
       'L': '\u2028',
-      'P': '\u2029'
+      'P': '\u2029',
```

Comment:
This file is compiled from https://github.com/connec/yaml-js/blob/579e6f6cc513ded1740959fc57aac3dc90c5f788/src/lib/scanner.coffee#L41. So the source file should be changed.

Reply:
Good call, thanks. The source file has been updated